deployments/ansible/roles/collector/README.md

```diff
@@ -22,7 +40,7 @@ how to use the role in a playbook with minimal required configuration:
 ```yaml
 - name: Install Splunk OpenTelemetry Connector
   hosts: all
-  become: yes
+  become_user: root
```

Comment:
Same comment here. Why changed? AFAIK `become_user: root` is default anyway, but it doesn't work if there is no `become: yes`.